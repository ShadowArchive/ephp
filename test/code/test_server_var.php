<?php

function data() {
    if (!isset($_SERVER["DATA"])) $_SERVER["DATA"] = 10;
    $default_server_values = array(
        'SERVER_SOFTWARE' => '',
        'REQUEST_URI' => '',
    );
    $_SERVER = array_merge( $default_server_values, $_SERVER );
}

$_SERVER['REQUEST_URI'] = '/server';
data();
data();
var_dump([$_SERVER['REQUEST_URI'], $_SERVER['DATA']]);

