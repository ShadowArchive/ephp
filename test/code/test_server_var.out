array(2) {
  [0]=>
  string(7) "/server"
  [1]=>
  int(10)
}
